HISTORY.rst

```diff
@@ -137,7 +144,7 @@ v0.20.0a1 (2019-02-13)
 * **breaking change** (maybe): more sophisticated argument parsing for validators, any subset of
   ``values``, ``config`` and ``field`` is now permitted, eg. ``(cls, value, field)``,
   however the variadic key word argument ("``**kwargs``") **must** be called ``kwargs``, #388 by @samuelcolvin
-* **breaking change**: Adds ``skip_defaults`` argument to ``BaseModel.dict()`` to allow skipping of fields that
+* **breaking change**: Adds ``skip_defaults`` argument to ``l.dict()`` to allow skipping of fields that
```

Comment:
I'm assuming this wasn't on purpose?

Reply:
Wow, how didn't I noticed this. Thanks for pointing that out!